grpc/cubic.proto

```diff
@@ -20,7 +20,7 @@ import "google/api/annotations.proto";
 import "google/protobuf/duration.proto";
 import "google/protobuf/empty.proto";
 
-option go_package = "cubicpb";
+option go_package = ".;cubicpb";
```

Comment:
What does  adding `.;` do ?

Reply:
go_package format is deprecated, it will soon need a full import path. this is a stop gap until we figure it out.